packages/compass-e2e-tests/helpers/commands/set-feature.ts

```diff
@@ -8,9 +8,11 @@ export async function setFeature(
   await browser.execute(
     async (_name, _value) => {
       // eslint-disable-next-line @typescript-eslint/no-var-requires
-      await require('electron').ipcRenderer.invoke('compass:save-preferences', {
-        [_name]: _value,
-      });
+      await require('compass-preferences-model').preferencesAccess.savePreferences(
+        {
+          [_name]: _value,
+        }
+      );
```

Comment:
Small drive-by fix to decouple this from the actual implementation of `savePreferences`

Reply:
Doesn't this rely on the fact that we are running the application inside the monorepo currently so the module can be resolved outside the bundled application code? In the real app this will fail with module not found, and we have plans to move the app outside the monorepo specifically so we can test that code is bundled with all required dependencies. 

If we don't want to rely on implementation details, I'd suggest exposing preferences globally in the renderer code itself (maybe only in test / dev mode), which is also handy when developing